documentation/src/main/asciidoc/topics/proc_adding_spring_cache.adoc

```diff
@@ -1,13 +1,31 @@
 [id='adding-spring-cache_{context}']
-= Adding Spring Cache support
+= Using {brandname} as a Spring Cache provider
 
-To add caching support to your application, with the following annotations:
+After adding {brandname} dependencies to your Spring application, you can enable caching by adding `@Cacheable` and `@CacheEvict` to select methods.
 
-* `@Cacheable` adds entries to the cache.
-* `@CacheEvict` removes entries from the cache.
+
+.Prerequisites
+
+* Add the {brandname} dependencies to your application project.
+* Create the required remote caches and configure Hot Rod client properties if you use a {brandname} Server deployment.
+
+[IMPORTANT]
+====
+
+For more information, see link:https://docs.spring.io/spring-framework/docs/current/reference/html/integration.html#cache-annotations-cacheable-default-key[Default Key Generation] in the Spring documentation.
+====
```

Comment:
This is probably my bad. We should remove the `[IMPORTANT]` admonition and move the link to a related info section at the end of the procedure.